src/ResourceManager/Network/Commands.Network/help/AzureRM.Network.md

```diff
@@ -86,6 +86,9 @@ Adds a route to a route table.
 ### [Add-AzureRmRouteFilterRuleConfig](Add-AzureRmRouteFilterRuleConfig.md)
 Adds a route filter rule to a route filter.
 
+### [Add-AzureRmServiceEndpointPolicyDefinition](Add-AzureRmServiceEndpointPolicyDefinition.md)
+{{Fill in the Synopsis}}
```

Comment:
Please fill in any missing synopsises in this file

Reply:
These are not my changes and beyond my purview. It seems someone added these files without checking in the changes to azurerm.network md.

Reply:
This is auto generated and beyond the scope of this PR - I am not sure what this command is.

Reply:
Looks like in the original repo the help files/cmdlets are there without network.md update or the other way round. This is causing this error - made this file have only changes related to this PR.